subresource-integrity/crossorigin-creds.js.sub.headers

```diff
@@ -0,0 +1,2 @@
+Access-Control-Allow-Origin: http://{{domains[]}}:{{ports[http][0]}}
```

Comment:
Maybe we should have a test where ACAO is set to the correct origin but allow creds isn't present and the request is cross-origin=anonymous?